lib/portlayer/network/network.go

```diff
@@ -48,8 +54,14 @@ func (e DuplicateResourceError) Error() string {
 }
 
 func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSource, sink extraconfig.DataSink) error {
-	var err error
-	initializer.Do(func() {
+	trace.End(trace.Begin(""))
+
+	initializer.once.Do(func() {
+		var err error
```

Comment:
There is a scoped error on line 79.  It doesn't look like a problem, but if you were to rely on this defer func to capture your err in side the for loop defined in on line 73 without seeing the shadow declaration on 79, that would be bad.  May want to rename that error on 79.